tlvf/yaml/tlvf/wfa_map/tlvApRadioBasicCapabilities.yaml

```diff
@@ -9,26 +9,22 @@ tlvApRadioBasicCapabilities:
     _type: eTlvTypeMap
     _value_const: TLV_AP_RADIO_BASIC_CAPABILITIES
   length: uint16_t
-  radio_uid: sMacAddr
+  radio_uid: sMacAddress
   maximum_number_of_bsss_supported: uint8_t
   operating_classes_info_list_length:
     _type: uint8_t
     _length_var: True
   operating_classes_info_list:
-    _type: sOperatingClassesInfo
+    _type: cOperatingClassesInfo
     _length: [ operating_classes_info_list_length ]
 
-sOperatingClassesInfo:
-  _type: struct
+cOperatingClassesInfo:
+  _type: class
   operating_class: uint8_t
   maximum_transmit_power_dbm: uint8_t
   statically_non_operable_channels_list_length:
     _type: uint8_t
     _length_var: True
   statically_non_operable_channels_list: 
-    _type: sChannel
-    _length: [ statically_non_operable_channels_list_length ]
-
-sChannel:
-  _type: struct
-  channel: uint8_t
+    _type: uint8_t
```

Comment:
I don't understand this change. I guess that means it should be in a separate commit with a proper explanation. And the rest of the changes in this file as well, I think.